app/controllers/schedule_controller.rb

```diff
@@ -43,6 +43,12 @@ def confirmation
 
   def schedule
     Edition.find_and_lock_current(document: params[:document]) do |edition|
+      if params[:review_status].nil?
```

Comment:
Could we synchronise the naming (`status` vs. `state`) here?

Reply:
Yep, i've gone with `review_status` for both scheduling and publishing controllers